orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/DeployManifestContext.java

```diff
@@ -17,88 +17,45 @@
 package com.netflix.spinnaker.orca.clouddriver.tasks.manifest;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.netflix.spinnaker.kork.artifacts.model.Artifact;
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Optional;
-import lombok.Getter;
-import org.jetbrains.annotations.Nullable;
+import javax.annotation.Nullable;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
 
-@Getter
-public class DeployManifestContext extends HashMap<String, Object> {
-  private final String source;
-  private final String manifestArtifactId;
-  private final Artifact manifestArtifact;
-  private final String manifestArtifactAccount;
-  private final Boolean skipExpressionEvaluation;
-  private final TrafficManagement trafficManagement;
-  private final List<String> requiredArtifactIds;
-  private final List<BindArtifact> requiredArtifacts;
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class DeployManifestContext extends ManifestContext {
+  @Nullable private List<String> manifests;
```

Comment:
When using the inline artifact editor, the manifests are actually converted to JSON before being stored in the stage, so I believe this needs to be `List<Object>`.

Reply:
👍